app/_includes/md/enterprise/cassandra-deprecation.md

```diff
@@ -1,7 +1,16 @@
+{% if include.length == "short" %}
+
+We don't recommend using Cassandra with {{site.base_gateway}},
+as support for Cassandra is [deprecated and planned to be removed](/gateway/{{include.kong_version}}/reference/configuration/#cassandra-settings).
```

Comment:
```suggestion
because support for Cassandra is [deprecated and planned to be removed](/gateway/{{include.kong_version}}/reference/configuration/#cassandra-settings).
```